aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -61,14 +61,29 @@ The following minimum changes are required to set up an existing ASP.NET Core pr
    For the sample app and command that follows, the service is:
 
    * Named **MyService**.
-   * Published to *c:\\svc* folder.
-   * Has an app executable named *AspNetCoreService.exe*.
+   * Published to the *c:\\svc* folder.
+   * Represented by an app executable named *AspNetCoreService.exe*.
 
    Open a command shell with administrative privileges and run the following command:
 
    ```console
    sc create MyService binPath= "c:\svc\aspnetcoreservice.exe"
    ```
+   
+   **Make sure the space is present between the `binPath=` argument and its value.**
```

Comment:
Since this is an important detail, I suggest putting it in an IMPORTANT widget to make it pop. If I were scanning the doc, I may miss this.

Reply:
There's a risk that readers read past them. I wish we had some hard data on it. 

I'll make that update when I get back, and something else has occurred to me. The use of the `--output` arg goes back to the original version of the topic. I'd rather only show one publish action here and just tell the reader briefly that they can use that arg and link over to it. I'm not feel'in two examples here upon reflection.